imgix/urlbuilder.py

```diff
@@ -140,35 +140,25 @@ def create_srcset(self, path, params=None):
         if not params:
             params = {}
 
-        width = params['w'] if 'w' in params else None
-        height = params['h'] if 'h' in params else None
-        aspect_ratio = params['ar'] if 'ar' in params else None
+        width = 'w' in params
+        height = 'h' in params
+        aspect_ratio = 'ar' in params
```

Comment:
Can we rename these variables to `has_width`, `has_height`, and `has_aspect_ratio`?
This is my bad for not doing it the first time around but I just noticed it now.

Reply:
Sure! I thought the same thing but didn't want to change too many things.